src/getGoogleToCozyStrategy.js

```diff
@@ -0,0 +1,36 @@
+const get = require('lodash/get')
+
+const { DOCTYPE_CONTACTS } = require('./constants')
+const getCozyMetadata = require('./getCozyMetadata')
+const transpiler = require('./transpiler')
+
+const getGoogleToCozyStrategy = (cozyUtils, SOURCE_ACCOUNT_ID) => ({
+  findRemoteDocument: (googleContact, cozyContacts) => {
+    return cozyContacts.find(
+      contact =>
+        get(contact, ['cozyMetadata', 'sync', SOURCE_ACCOUNT_ID, 'id']) ===
+        googleContact.resourceName
+    )
+  },
+
+  shouldSave: (googleContact, cozyContact) => {
+    return cozyContact === undefined
+  },
+
+  save: googleContact => {
+    const newCozyContact = transpiler.toCozy(googleContact)
+    const newCozyContactWithMetadata = {
+      _type: DOCTYPE_CONTACTS,
+      ...newCozyContact,
+      ...getCozyMetadata(newCozyContact, googleContact, SOURCE_ACCOUNT_ID)
+    }
+
+    return cozyUtils.save(newCozyContactWithMetadata)
+  },
+
+  afterSave: (googleContact, cozyContact) => {
+    return Promise.resolve(cozyContact.id)
```

Comment:
Nit: I think we can simply return `cozyContact.id`, no need to wrap it in a promise, even if we `await` on the other side?

Reply:
it seems ok!